lib/src/presentation/src/presentation_widget.dart

```diff
@@ -1,7 +1,6 @@
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 import 'package:prides/prides.dart';
-
 import 'package:prides/src/slide/widgets/end_slide.dart';
```

Comment:
New line is needed here
```suggestion

import 'package:prides/src/slide/widgets/end_slide.dart';
```